flexget/utils/imdb.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
         self.min_diff = 0.01
         self.debug = False
 
-        self.max_results = 10
+        self.max_results = 50
```

Comment:
you can add the option to `imdb_lookup`. that can use [partial](https://docs.python.org/2/library/functools.html#functools.partial) when it calls `imdb.py` operations.